src/snippet.rs

```diff
@@ -375,8 +379,16 @@ impl<'a> Patch<'a> {
     }
 }
 
-/// The location of the [`Snippet`] (e.g. a path)
-#[derive(Clone, Debug)]
+/// The location of the [`Snippet`] (e.g. a path).
+///
+/// This should be used if you want to display the file details by
+/// itself, or you want to set the line number and column explicitly.
+/// Otherwise, it is better to use [`Snippet::origin`].
```

Comment:
This should be clarified that you mean that this is a parallel item to `Snippet` (can't remember all of the terms and not digging in to provide a suggestion atm)